test/xpu_api/numerics/complex.number/cmplx.over/arg.pass.cpp

```diff
@@ -48,9 +48,9 @@ ONEDPL_TEST_NUM_MAIN
     test<float>();
     IF_DOUBLE_SUPPORT(test<double>())
     IF_LONG_DOUBLE_SUPPORT(test<long double>())
-    test<int>();
-    test<unsigned>();
-    test<long long>();
+    IF_DOUBLE_SUPPORT(test<int>();
```

Comment:
In all these cases type ```double``` used inside test.

Reply:
What is the difference between this case and other cases when a type of literals is changed from ```double``` to ```float```? Can we use ```float``` type here as well?

Reply:
For ```float``` type we made call ```test<float>();``` above in line 48.

Reply:
I mean these lines:

```c++
    static_assert((std::is_same<decltype(dpl::arg(x)), double>::value), "");
    assert(dpl::arg(x) == dpl::arg(dpl::complex<double>(static_cast<double>(x), 0)));
```

Reply:
Discussed offline.